apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/ErrorReporting/ReportEventsTo.cs

```diff
@@ -22,7 +22,7 @@ namespace Google.Cloud.Diagnostics.Common
     /// <summary>
     /// The location error events will be sent to.
     /// </summary>
-    public enum ReportEventsToLocation
+    public enum EventTarget
```

Comment:
So my rational for not changing the class was like Mike said ReportEventsTo.Logging(...) is easy to parse.

Users will almost exclusivly interact at the class level ReportEventsTo.Logging(...).  The enum more of less internal.  That being said I'd rather have a proper name than not